### PR TITLE
Add config options back to 1.22 documentation

### LINKED ISSUE #3983: Missing admin/config page for 1.22
_404 for Telegraf 1.22 admin/config page_

Check the installation page, linked below, for the text "For more advanced configuration details, see the [configuration documentation](https://docs.influxdata.com/telegraf/v1.22/administration/configuration/)."

##### Relevant URLs
- [ Broken Telegraf 1.22 admin config page ](https://docs.influxdata.com/telegraf/v1.22/administration/configuration/)
- [ Referring page - Telegraf 1.22 Installation](https://docs.influxdata.com/telegraf/v1.22/install/)
 

##### What products and version are you using?

Telegraf 1.22

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #3983, #3955, #3947

### CODE REVIEW

File: content/telegraf/v1.22/get_started.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ After you've [downloaded and installed Telegraf](/telegraf/v1.22/install/), you'
 
 ## Configure Telegraf
 
-Define which plugins Telegraf will use in the configuration file. Each configuration file needs at least one enabled [input plugin](/telegraf/v1.22/plugins/inputs/) (where the metrics come from) and at least one enabled [output plugin](/telegraf/v1.22/plugins/outputs/) (where the metrics go).
+Define which plugins Telegraf will use in the configuration file. Each configuration file needs at least one enabled [input plugin]((/{{< latest "telegraf" >}}/plugins/inputs/) (where the metrics come from) and at least one enabled [output plugin]((/{{< latest "telegraf" >}}/plugins/outputs/) (where the metrics go).
```
Comment (lwandzura) [inline content/telegraf/v1.22/get_started.md:18]: should be (/{{< latest "telegraf" >}} - think you have an extra beginning parenthesis in a couple of places

File: content/telegraf/v1.22/get_started.md [hunk 2]
```diff
@@ -15,12 +15,12 @@ After you've [downloaded and installed Telegraf](/telegraf/v1.22/install/), you'
...
 
 The following example generates a sample configuration file with all available plugins, then uses `filter` flags to enable specific plugins.
 
 {{% note %}}
-For details on `filter` and other flags, see [Telegraf commands and flags](/telegraf/v1.22/commands/).
+For details on `filter` and other flags, see [Telegraf commands and flags]((/{{< latest "telegraf" >}}/commands/).
```
Comment (lwandzura) [inline content/telegraf/v1.22/get_started.md:23]: should be (/{{< latest "telegraf" >}}

**Review by lwandzura — CHANGES_REQUESTED** [review]
just a couple of fixes

File: content/telegraf/v1.22/configuration.md [hunk 3]
```diff
@@ -0,0 +1,474 @@
...
+  telegraf_1_22_ref:
+
+    name: Configuration options
+    weight: 20
+
+---
+
+The Telegraf configuration file (`telegraf.conf`) lists all available Telegraf plugins. See the current version here: [telegraf.conf](https://github.com/influxdata/telegraf/blob/master/etc/telegraf.conf).
+
+> To quickly get started with Telegraf, see [Get started](/{{< latest "telegraf" >}}/get_started/).
```
Comment (sanderson) [inline content/telegraf/v1.22/configuration.md:14]: @noramullen1 @lwandzura We shouldn't use the `{{< latest >}}` shortcode when linking to content in the same product. Only use it when linking to other product content. When we release Telegraft 1.23, all the latest links in 1.22 would point to 1.23 and shift the version context when users click on the link. We don't want that.

Comment (noramullen1) [reply content/telegraf/v1.22/configuration.md]: oh duh, covid brain

**Pull request merged**